site/layouts/_default/docs.html

```diff
@@ -1,6 +1,6 @@
 {{ define "body_override" }}<body{{ if (eq .Page.Params.toc true) }} data-bs-spy="scroll" data-bs-target="#TableOfContents"{{ end }}>{{ end }}
 {{ define "main" }}
-  <div class="container-fluid container-max-width bd-gutter mt-3 my-md-4 bd-layout">
+  <div class="container-fluid container-max-width bd-gutter mt-medium my-md-tall bd-layout">
```

Comment:
This layout sounds a little bit too compact. It was pretty packed already in Boosted docs. Let's check with the designers.

Reply:
Let's wait for https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2754#discussion_r1822083985 final values. Could be fixed automatically.